### ISSUE TITLE
Correction & Annotation: Always 1 more sentence shown than configured in settings

### ISSUE THREAD

**Issue by reckart** [webanno/webanno#10]
```
What steps will reproduce the problem?
1. Document: conll2002 format
2. Open an annotation or correction document
3. Change in the view settings: number of sentences: 1

What is the expected output?
Only 1 sentence displayed for annotation

What do you see instead?
2 sentences displayed for annotation


What version of the product are you using? On what operating system?
webanno.war 2.1.1 (2015-04-07 22:51:47, build 2574) on Ubuntu 14.4

Please provide any additional information below.

Not always, but sometimes also reproducible with the demo jar.


Currently going back to 2.1.0, because this fixes it for us (simple downgrade by swapping
the war file).
```

Original issue reported on code.google.com by `tobias.kroenke` on 2015-05-12 17:37:11

**Comment by reckart** [comment]
Should be fixed by b203c219f022ef34224acc02d8c6a1d873449372.

**Comment by reckart** [comment]
Added to 2.1.x branch: 6993187050be32b60d4827cdb4b8a105c66fca43.
